src/components/login/ProviderLink.tsx

```diff
@@ -11,15 +11,24 @@ interface Props {
 }
 
 function loginMethod(provider: LoginProvider) {
+  switch (provider.method) {
+    case provider.method:
+      return provider.name
+
+    default:
+      break;
+  }
+}
+
+function loginButton(provider: LoginProvider) {
+  const logoPath = "/img/login/"
+
   switch (provider.method) {
     case 'github':
-      return 'GitHub'
+      return logoPath + "github-icon.png"
 
     case 'gitlab':
-      return 'GitLab'
-
-    case 'google':
```

Comment:
Not sure I'm up to date, no google for now?

Reply:
We're still using Google, there just hasn't been a logo used yet. I'll try to find one which is a SVG and add it in.